azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java

```diff
@@ -27,21 +27,21 @@
 	 * The returned CompletableFuture completes with true if the checkpoint store exists or false if it
 	 * does not. It completes exceptionally on error.
 	 * 
-	 * @return CompletableFuture -> true if it exists, false if not
+	 * @return CompletableFuture -&gt; true if it exists, false if not
```

Comment:
I've used the @literal tag in the codebase. We should decide if we want to use @literal or HTML character codes. I don't have a preference either way, we should just be consistent.

Reply:
I don't have a preference either. I've tended to use character codes in my past, but I don't prefer one over the other. Feel free to decline this PR if you'd rather use literal tags instead.

Reply:
@sjkwak @SreeramGarlapati @JamesBirdsall

Reply:
The main thing I want to see is that as part of the commit process we start to standardise on ensuring that `mvn javadoc:javadoc` completes without error. This way we can ensure that the repo is always able to generate API docs (which I am trying to do on a daily basis here: https://github.com/JonathanGiles/azure-javadocs)

Reply:
Absolutely agree. The plan is to have Travis ensure a `mvn clean package` builds for javadocs and code before a PR can be merged.